chart/templates/drupal-deployment.yaml

```diff
@@ -33,13 +33,6 @@ spec:
         livenessProbe:
           tcpSocket:
             port: 9000
-        readinessProbe:
-          exec:
-            command:
-            - /bin/sh
-            - -c
-            - if [ ! {{ include "drupal.deployment-in-progress-test" . }} ]; then timeout -t 3 drush check-bootstrap; else exit 1; fi
```

Comment:
I believe that test is still needed so we don't get the tables created while post-deployment doing initial `drush site-install`.

Reply:
You are right. We can skip `drush check-bootstrap` but should still check for the deployment in progress.